src/utils/d2-api.ts

```diff
@@ -11,3 +20,20 @@ export function getMajorVersion(version: string): number {
 export function getD2APiFromInstance(instance: Instance) {
     return new D2Api({ baseUrl: instance.url, auth: instance.auth, backend: "fetch" });
 }
+
+export const getMetadataByKey = async ({ api, key }: ApiType): Promise<{ [key: string]: unknown }[]> => {
+    const { response } = api.dataStore("glass").getMetadata(key);
+    const responseData = (await response()).data;
+    const data = responseData?.value && JSON.parse(responseData?.value);
+
+    return data;
+};
+
```

Comment:
We should always to use the domain previously to write data or get data from the dhis2 (data store)